test/TestLoadContract.js

```diff
@@ -0,0 +1,39 @@
+var assert = require('assert');
+const from_address = "0x90f8bf6a479f320ead074411a4b0e7944ea8c9c1"
+
+var loadContract = require("../functions/loadContract.js")
+
+describe('loadContract', function() {
+  describe('weird test', function() {
+    it('should do stuff without throwing', function() {
+		var ProductionRegistry = loadContract("ProductionRegistry")
+		var Production = loadContract("Production")
+
+		var prodCount;
+		var prodReg
+
+		ProductionRegistry.deployed().then(function(prodRegInstance) {
+			prodReg = prodRegInstance
+			return prodReg.getProductionsCount.call()
+		}).then(function(result) {
```

Comment:
Was just a test to do contract tests ? If not it might be good to follow the setup->exercice->verify (like in [https://github.com/peer2print/contracts/blob/master/test/TestProduction.js#L94])